nodewatcher/nodewatcher.py

```diff
@@ -154,35 +169,46 @@ def main():
     log.info("nodewatcher startup")
     instance_id = getInstanceId()
     hostname = getHostname()
-    region, asg_name, scheduler, proxy_config = getConfig(instance_id)
+    region, asg_name, scheduler, proxy_config, idle_time = getConfig(instance_id)
+
 
     s = loadSchedulerModule(scheduler)
 
+    if os.path.isfile(_NW_IDLETIME_FILE):
+        data = json.loads(open(_NW_IDLETIME_FILE).read())
+    else:
+        data = {'idle_time': 0}
+
+    atexit.register(saveIdleTime, data)
     while True:
         time.sleep(60)
-        ec2_conn = boto3.resource('ec2', region_name=region, config=proxy_config)
         asg_conn = boto3.client('autoscaling', region_name=region, config=proxy_config)
-        hour_percentile = getHourPercentile(instance_id, ec2_conn)
-        log.info('Percent of hour used: %d' % hour_percentile)
-
-        if hour_percentile < 95:
-            continue
 
         jobs = getJobs(s, hostname)
-        if jobs == True:
+        if jobs:
             log.info('Instance has active jobs.')
```

Comment:
Add %s for logging `jobs`

Reply:
jobs is a boolean, why do we want to log it?

Reply:
No, the function should be better named as `hasJobs` but in another patch, ok.

Reply:
Okay, that I can do!